update.sh

```diff
@@ -1145,7 +1161,7 @@ print_intro() {
     echo "+------------------------------------------------+"
     echo "| UCRM - Complete WISP Management Platform       |"
     echo "|                                                |"
-    echo "| https://ucrm.ubnt.com/         (updater v2.11) |"
+    echo "| https://ucrm.ubnt.com/          (updater v3.0) |"
```

Comment:
Is this related to UCRM version, or is it not? Magic number?

Reply:
not related